Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM aica-technology/ros-ws:noetic
+
+RUN sudo apt-get update && sudo apt-get install -y ros-noetic-turtlesim && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /home/${USER}/ros_ws
+COPY --chown=${USER} ./turtle_example ./src/turtle_example
```

Comment:
Maybe add the `catkin_make` step here just so it already creates the `devel/lib/python3` folder